vault/secret_test.go

```diff
@@ -10,32 +10,42 @@ import (
 	"github.com/influxdata/influxdb/v2"
 	influxdbtesting "github.com/influxdata/influxdb/v2/testing"
 	"github.com/influxdata/influxdb/v2/vault"
-	testcontainer "github.com/testcontainers/testcontainers-go"
+	testcontainers "github.com/testcontainers/testcontainers-go"
 )
 
 func initSecretService(f influxdbtesting.SecretServiceFields, t *testing.T) (influxdb.SecretService, func()) {
 	token := "test"
 	ctx := context.Background()
-	vaultC, err := testcontainer.RunContainer(ctx, "vault", testcontainer.RequestContainer{
-		ExportedPort: []string{
-			"8200/tcp",
+	vaultC, err := GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: ContainerRequest{
```

Comment:
Missing package reference:
```suggestion
		ContainerRequest: testcontainers.ContainerRequest{
```